contracts/mvx-game-sc/tests/game_sc_blackbox_tests.rs

```diff
@@ -60,17 +70,44 @@ impl GameContractState {
                         .nonce(3)
                         .balance(BALANCE)
                         .esdt_balance(TOKEN_ID, BALANCE),
+                )
+                .put_account(
+                    USER3_ADDR,
+                    Account::new()
+                        .nonce(4)
```

Comment:
Starting nonces should be `1` for each account so you can more easily compute nonces with executed transactions.